app/mailers/asset_notification.rb

```diff
@@ -1,22 +1,38 @@
 class AssetNotification < ActionMailer::Base
   default from: Alonetone.email
 
-  def upload_notification(asset, email, _sent_at = Time.now)
-    @track = asset.name
-    @description = asset.description
-    @name = asset.user.name
-    @user = asset.user
-    @title = asset.title.present? ? asset.title : "new track"
+  def upload_notification(asset, follower_email, _sent_at = Time.now)
+    @asset_author = asset.user
```

Comment:
I understand wanting to get away from the ambiguous `@user` in situations like this. How do you feel about `@artist` or `@musician`?